lib/ex_doc/retriever.ex

```diff
@@ -458,6 +464,25 @@ defmodule ExDoc.Retriever do
     {id, id}
   end
 
+  defp module_group(module, group_patterns) do
+    Enum.find_value(group_patterns, fn {group, patterns} ->
+      patterns = List.wrap patterns
+      module_in_patterns(module, patterns) && Atom.to_string(group)
+    end)
+  end
+
+  defp module_in_patterns(module, patterns) do
+    "Elixir." <> module_string = Atom.to_string module
+
+    Enum.any?(patterns, fn pattern ->
+      case pattern do
+        %Regex{} = regex -> Regex.match?(regex, module_string)
+        string when is_binary(string) -> module_string === string
+        atom -> atom === module
```

Comment:
Please use `==` and above. :)

Reply:
I can certainly do this. I'm just curious, how would/should one decide between `==` and `===`?